src/agenda/reservation-list/index.tsx

```diff
@@ -246,7 +245,7 @@ class ReservationList extends Component<ReservationListProps, ReservationsListSt
     const dateIsSame = sameDate(day, this.selectedDay);
     if (!dateIsSame && this.scrollOver) {
       this.selectedDay = day.clone();
-      invoke(this.props, 'onDayChange', day.clone());
+      this.props.onDayChange?.(new Date(day.getUTCMilliseconds()));
```

Comment:
why changing from `day.clone()` to `new Date(day.getUTCMilliseconds())`?

Reply:
type issue, but seams the type on the prop is wrong so I'll change it again

Reply:
Changed the type on the onDayChange prop